### ISSUE TITLE
Need Help with Nesting my platforms

### ISSUE THREAD

**Issue by anupchavda1** [nfarina/homebridge#2173]
Hi there
Been using this plugin for a long time and everything works really well, except i tried to add 2 plugins and now i can't figure out the nesting, and was hoping someone could help me?

`{
	"bridge": {
		"name": "HomebridgeNest",
		"username": "xx:xx:xx:xx:xx:xx,
		"port": 35565,
		"pin": "032-64-155"
	},

	"description": "Homebridge for Nest automation.",

	"accessories": [{
			"accessory": "DenonMarantzAVR",
			"name": "Amplifier",
			"ip": "192.168.1.xx",
			"defaultInput": "MPLAY",
			"defaultVolume": 35,
			"minVolume": 10,
			"maxVolume": 75,
			"doPolling": true,
			"pollingInterval": 60
		}
	],

	"platforms": [		
	    {
			"platform": "Nest",

			"token": "hidden",

			"clientId": "hidden",
			"clientSecret": "hidden",
			"code": "hidden"
		},
		{
      "platform": "config",
      "name": "Config",
      "port": 8080,
      "sudo": true
        },
        {
			"platform": "FoscamCamera",
			"name": "Homebridge Camera",
			"cameras": [{
				"username": "hidden",
        			"password": "hidden",
       				"host": "192.168.1.xx",
        			"port": 88,
  				"videoConfig": {
					"source": "-re -i rtsp://hidden:hidden@192.168.1.xx:88/videoMain",
					"stillImageSource": "-i http://192.168.1.hidden:88//cgi-bin/CGIProxy.fcgi?cmd=snapPicture2&amp;usr=hidden&amp;pwd=hidden",					
					"maxStreams": 2,
					"maxWidth": 1280,
					"maxHeight": 720,
					"maxFPS": 30,
					"maxBitrate": 200,
        				"vcodec": "h264_omx",
       					"audio": false,
        				"packetSize": 188
						}
		},
		{ "platform"  : "ring-video-doorbell",
		  "name"      : "Doorbell",
 		  "username"  : "hidden",
 		  "password"  : "hidden", 
		  "options"   : { "retries": 5, "ttl": 5, "verboseP" : false },
		  "ringing"   : { "event": "single", "motion": false, "contact": false }
},

		{
			"platform": "BroadlinkRM",
			"name": "Broadlink RM",
			"accessories": [{
					"name": "Living Room TV",
					"type": "switch",
					"pingIPAddress": "192.168.1.x",
					"pingFrequency": 5,
					"pingIPAddressStateOnly": true,
					"host": "192.168.1.x",
					"data": {
						"on": "2600d200939512381138123811141114111312131213113912371238111411141113121311141114113911131213111411141114111411381213113911381139113812381139110006009495123711391138111411141114111411131238113911381213111411141113121311141139111312131114111411141114113811141139113811391435123811381200060093951238113812381114111411131213121311391138113614141114111312131114111411381213111411141114111312131139111411381139113812381138123811000d05000000000000",
						"off": "2600d200939512381138123811141114111312131213113912371238111411141113121311141114113911131213111411141114111411381213113911381139113812381139110006009495123711391138111411141114111411131238113911381213111411141113121311141139111312131114111411141114113811141139113811391435123811381200060093951238113812381114111411131213121311391138113614141114111312131114111411381213111411141114111312131139111411381139113812381138123811000d05000000000000"
					}
				},
				{
					"name": "AirCon",
					"type": "switch",
					"host": "192.168.1.x,
					"data": {
						"on": "2600400000012286123212101210111111321211111012101111111111111111111111111111111111111132111111321111113211111111113212101210113212000d050000000000000000",
						"off": "2600400000012285133112111111111111321210111111101232113211111111111111111111111111101111111111111110123211111133111111111111113211000d050000000000000000"
					}
				}

			]
		}
	]
}]}

`

Basically, everything after the camera is not being recognised as a platform althoug Homekit is running and i can control everything up to and including the camera.

Any help would be gratefully appreciated!

**Comment by Mart-555** [comment]
You have the line :    "cameras": [{
But I cannot see a closing ] at the end of the camera info to match with the opening [

**Comment by Mart-555** [comment]
In fact I think you are also missing a } symbol in order to fully close the cameras entry.
